Utilities/import-llvm

```diff
@@ -190,6 +235,51 @@ def main():
 
     include_dir = os.path.join(sourcekit_srcroot, 'include')
 
+    # Create Symlinks for SwiftPM include directory resolution
+    os.makedirs(os.path.join(sourcekit_llvm_lib, 'Support', 'include', 'llvm'))
+    os.makedirs(os.path.join(sourcekit_llvm_lib, 'Demangle', 'include'))
+    os.makedirs(os.path.join(sourcekit_llvm_lib, 'include'))
+    os.makedirs(os.path.join(sourcekit_llvm_include, 'Demangle', 'include'))
+    os.symlink(os.path.join('..', '..', '..', '..', '..', 'include', 'llvm', 'ADT'),
+               os.path.join(sourcekit_llvm_lib, 'Support', 'include', 'llvm', 'ADT'))
+    os.symlink(os.path.join('..', '..', '..', '..', '..', 'include', 'llvm', 'Config'),
+               os.path.join(sourcekit_llvm_lib, 'Support', 'include', 'llvm', 'Config'))
+    os.symlink(os.path.join('..', '..', '..', '..', '..', 'include', 'llvm', 'Support'),
+               os.path.join(sourcekit_llvm_lib, 'Support', 'include', 'llvm', 'Support'))
+    os.symlink(os.path.join('..', '..', '..', '..', 'include', 'llvm-c'),
+               os.path.join(sourcekit_llvm_lib, 'Support', 'include', 'llvm-c'))
+    os.symlink(os.path.join('..', '..', '..', '..', 'include', 'llvm'),
+               os.path.join(sourcekit_llvm_lib, 'Demangle', 'include', 'llvm'))
+    os.symlink(os.path.join('..', '..', '..', 'llvm'),
+               os.path.join(sourcekit_llvm_include, 'Demangle', 'include', 'llvm'))
+    os.symlink(os.path.join('..', '..', '..', 'include', 'llvm'),
+               os.path.join(sourcekit_llvm_lib, 'include', 'llvm'))
+    os.symlink(os.path.join('..', '..', '..', 'include', 'llvm-c'),
+               os.path.join(sourcekit_llvm_lib, 'include', 'llvm-c'))
+
+    copyfile(os.path.join(sourcekit_srcroot, 'Utilities', 'import-llvm.d', 'lib', 'LLVMSupport', 'include', 'module.modulemap'),
+             os.path.join(sourcekit_srcroot, 'lib', 'LLVMSupport', 'include', 'module.modulemap'))
+
+    copyfile(os.path.join(sourcekit_srcroot, 'Utilities', 'import-llvm.d', 'include', 'llvm', 'ADT', 'OptionSet.h'),
+             os.path.join(sourcekit_srcroot, 'include', 'llvm', 'ADT', 'OptionSet.h'))
+
+    # AArch64TargetParser uses a relative path to find the .def file, so we
+    # copy this one to the expected location :(
+    def_location = os.path.join(sourcekit_srcroot, 'lib', 'include', 'llvm', 'Support')
+    if os.path.exists(def_location):
+        shutil.rmtree(def_location)
+    os.makedirs(def_location)
+    copyfile(os.path.join(sourcekit_srcroot, 'include', 'llvm', 'Support', 'AArch64TargetParser.def'),
+             os.path.join(def_location, 'AArch64TargetParser.def'))
+
+    # Process.cpp uses LLVM_ENABLE_CRASH_DUMPS which is usually set in CMake.
```

Comment:
This should be in config.h.  In general, it would be good to look at a new config.h from an llvm build, although it will need to be merged manually with what's here, which is already customized.  But that more general change doesn't need to happen in this PR.